docs/cellxgene_census_FAQ.md

```diff
@@ -0,0 +1,109 @@
+# Frequently Asked Questions for CZ CELLxGENE Discover Census
+
+Last updated: Apr, 2023.
+
+- [Why should I use the Census?](#Why-should-I-use-the-Census)
+- [Why does the Census not have a normalized layer or embeddings?](#Why-does-the-Census-not-have-a-normalized-layer-or-embeddings)
+- [How does the Census differentiate from other services?](#How-does-the-Census-differentiate-from-other-services?)
+- [Can I query human and mouse data in a single query?](#Can-I-query-human-and-mouse-data-in-a-single-query)
+- [Where are the Census data hosted?](#Where-are-the-Census-data-hosted)
+- [How can I increase the performance of my queries?](#How-can-I-increase-the-performance-of-my-queries)
+- [Can I use conda to install the Census Python API?](#Can-I-use-conda-to-install-the-Census-Python-API)
+- [How can I ask for support?](#How-can-I-ask-for-support)
+- [How can I ask for new features?](#How-can-I-ask-for-new-features)
+- [How can I contribute my data to the Census?](#How-can-I-contribute-my-data-to-the-Census)
+- [Why do I get an `ArraySchema` error when opening the Census?](#Why-do-I-get-an-ArraySchema-error-when-opening-the-Census)
```

Comment:
A few more possible FAQs:
"May I (How do I) cite Census data in a publication?"
"What are the terms of use for the data?"
"Can I retrieve the original H5AD datasets from which the Census was built?"
"Is there a summary of the data contained in the Census?" (link to notebook showing how retrieve summary info)
"A new dataset is available in CZ Discover, but I don't see it in the Census.  Why?" (We update once per week)

Reply:
"May I (How do I) cite Census data in a publication?" - use https://cellxgene.cziscience.com/docs/08__Cite%20cellxgene%20in%20your%20publications until we have embedded citations?